src/Protocol/DFRN.php

```diff
@@ -2807,15 +2807,13 @@ private static function processDeletion($xpath, $deletion, $importer)
 			}
 		}
 
-		$entrytype = self::getEntryType($importer, $item);
-
-		if (!$item["deleted"]) {
-			logger('deleting item '.$item["id"].' uri='.$uri, LOGGER_DEBUG);
-		} else {
+		if ($item["deleted"]) {
 			return;
 		}
 
-		Item::deleteById($item["id"]);
+		logger('deleting item '.$item["id"].' uri='.$uri, LOGGER_DEBUG);
+
+		Item::delete(['id' => $item["id"]]);
```

Comment:
Code standards: Please use a consistent quoting style in the same line at least.